README.md

```diff
@@ -1,51 +1,27 @@
 # Vipps QR API v1
```

Comment:
Technically we are now on version 2. I'm not sure how to write it.
Either remove version numbers or a chapter for each version.

@khenrichsen any, thoughts?

Reply:
@Sigvedash - in the other APIs, they usually  write it like this:
- Vipps eCommerce API version 2
- Vipps PSP API version 3

So, we can change this to "Vipps QR API version 2".